.gitignore

```diff
@@ -37,4 +37,4 @@ project.fragment.lock.json
 backup/**/*.*
 .dotnet/**
 global.json
-korebuild-lock.txt
```

Comment:
Intentional?

Reply:
Yup. We want builds to be reproducible, so we've started committing exact package and tools versions to source, both in dev and rel/2.0.x branches.